wavefront/time_uniform_sample.go

```diff
@@ -0,0 +1,248 @@
+package wavefront
+
+import (
+	"sync"
+	"time"
+
+	metrics "github.com/rcrowley/go-metrics"
+	"istio.io/istio/pkg/log"
+)
+
+// TimeUniformSample is a Sample to store valuse only for a specific time
+type TimeUniformSample struct {
+	mutex    sync.Mutex
+	values   Queue
+	lifetime time.Duration
+}
+
+// list of all samples created to be cleaned periodically
+var sampleList = make([]*TimeUniformSample, 0)
+
+// recycle ticker that will fire a samples cleaning
+var ticker *time.Ticker
+
+// time delay for the recycle ticker
+const recycleTime = time.Second * 5
```

Comment:
Up to you, but I generally place configurable `const`s right after the `import` statement at the top.